style.css

```diff
@@ -0,0 +1,73 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap');
+
+* {
+  font-family: 'Poppins', sans-serif;
+  box-sizing: border-box;
+}
+
+body {
+  background-image: url(Images/header-bg.png);
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  width: 375px;
+  height: 718px;
+  left: 0;
+  top: 0;
+  background-repeat: no-repeat;
+  margin: 0;
+}
+
+.head-section {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 12px;
+  padding: 115px 24px 115px 24px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: 100%;
+}
+
+.menuIcone {
+  position: absolute;
+  left: 80.67%;
+  right: 50.67%;
+  top: 5.67%;
+  bottom: 5.67%;
+}
+
+header {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.connect {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  text-transform: uppercase;
+  font-size: 16px;
+  font-weight: 500;
+  color: rgb(61, 81, 255);
+  margin: 0;
+}
+
+.social-icon ul {
+  width: 170px;
+  padding: 0;
+  margin: 0;
+  display: flex;
+  align-items: flex-start;
+  justify-content: space-between;
+  list-style: none;
+}
```

Comment:
- [ ] kindly consider fixing your design which will make your design an exact duplicate of the provided design
- your design 👇 
<img width="427" alt="Screen Shot 2023-03-30 at 01 52 59" src="https://user-images.githubusercontent.com/60146030/228693717-ad4bc1ec-fbcd-4a90-a2ff-48d03d11376c.png">
 
- provided design 👇 
<img width="372" alt="Screen Shot 2023-03-30 at 01 55 53" src="https://user-images.githubusercontent.com/60146030/228694174-e1f51feb-773c-4d62-ae84-5bcf95fd5d88.png">
 consider making sure each section has the margins (top, bottom, left, right) defined in the design.this will help you to have a wonderful website. and again put into consideration adding that bottom left radius as shown in the above provided design to have an exact duplicate of the provided design